### PR TITLE
fix - Add Docker versioning

### PR DESCRIPTION
**Pull request by mikhaelsantos**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,19 +10,25 @@ branches:
...
+  - stage: Build docker (Dev/Nightly)
     if: tag IS blank
     install: true
-    env:
-    - OWNER=streamr
-    - IMAGE_NAME=streamr-ganache
-    - TAG=dev
     script:
-    - docker build -t $OWNER/$IMAGE_NAME:$TAG .
+    - docker build -t $OWNER/$IMAGE_NAME:taggit .
```
Comment (mirpo) [inline .travis.yml:23]: taggit? what is it? why not `nightly`?

File: .travis.yml [hunk 2]
```diff
@@ -10,19 +10,25 @@ branches:
...
     deploy:
     - provider: script
-      script: bash .travis_scripts/deploy_docker.sh staging
\ No newline at end of file
+      script: bash .travis_scripts/deploy_docker.sh dev
+  - stage: Build docker (Production)
+    if: tag IS present
+    install: true
+    script:
+    - docker build -t $OWNER/$IMAGE_NAME:taggit .
```
Comment (mirpo) [inline .travis.yml:31]: why no `tag` here ?

**Review by mirpo — CHANGES_REQUESTED** [review]
LGFM, just don't understand `taggit`

File: .travis.yml (hunk 2 above)
Comment (mikhaelsantos) [reply .travis.yml:31]: It is just a place holder name when building taggit aka still need to tag correctly. The docker script will then do the needed tagging.
`    
docker tag "$OWNER/$IMAGE_NAME:taggit" "$OWNER/$IMAGE_NAME:$TRAVIS_TAG"
docker tag "$OWNER/$IMAGE_NAME:taggit" "$OWNER/$IMAGE_NAME:latest"
`

File: .travis.yml (hunk 1 above)
Comment (mikhaelsantos) [reply .travis.yml:23]: Check the other file, it contains all the tagging logic

**Review by jtakalai — APPROVED** [review]
looks good

**Review by harbu — APPROVED** [review]

**Pull request merged**